com.unity.render-pipelines.core/Editor/LookDev/DisplayWindow.cs

```diff
@@ -680,6 +681,16 @@ void OnGUI()
                     rootVisualElement.styleSheets.Add(styleSheetLight);
             }
 
+            // [case 1245086] Guard in case the SRP asset is set to null (or to a not supported SRP) when the lookdev window is already open
+            if (!LookDev.supported && OnUpdateRequestedInternal !=null)
```

Comment:
I think the OnUpdateRequestedInternal check is not needed here. If we reach this code, it means that we had a working SRP asset and we are rendering with the editor camera. So simply checking that the LookDev is not supported anymore should be enough.

( OnUpdateRequestedInternal should be set as long as the LookDev windows is open because the Compositor rely on it. )

[optional] replace by: if (!LookDev.supported)

Reply:
When the editor was reloading (after a code edit) with the lookdev window open, I was hitting this spot for 2 frames with a null SRP and null OnUpdateRequestedInternal. So I have added the OnUpdateRequestedInternal check to avoid printing the error (and closing the window) in this case.

Reply:
I have added a comment for future reference.